sunbird-mentoring/controllers/mentoring.js

```diff
@@ -43,12 +43,283 @@ const profileRead = async (req, res, selectedConfig) => {
 	}
 }
 
+const createUser = async (req, res, responses, selectedConfig) => {
```

Comment:
@rakeshSgr Let's remove all console logs, we can otherwise put it inside an IF condition which will be enabled if from env DEBUG_MODE = true

Reply:
resolved